spec/serializer_spec.rb

```diff
@@ -460,21 +485,21 @@ def serialize_primary(object, options = {})
       expect(actual_data['included']).to eq(expected_data['included'])
       expect(actual_data).to eq(expected_data)
     end
-    it 'includes linkage in compounded resources only if the immediate parent was also included' do
+    it 'includes data in compounded resources only if the immediate parent was also included' do
       comment_user = create(:user)
       long_comments = [create(:long_comment, user: comment_user)]
       post = create(:post, :with_author, long_comments: long_comments)
 
       expected_primary_data = serialize_primary(post, {
         serializer: MyApp::PostSerializer,
-        include_linkages: ['long-comments'],
+        include_data: ['long-comments'],
```

Comment:
For clarity since this is just an internal variable name, can you rename `include_data` to `include_relationships` here and everywhere?